ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activityAnswer.component.ts

```diff
@@ -13,7 +16,7 @@ import { BlockVisibility } from '../../models/activity/blockVisibility';
                                  [readonly]="readonly"
                                  (valueChanged)="onChange($event)">
     </ddp-activity-boolean-answer>
-    <ddp-activity-text-answer *ngIf="isTextQuestion(block) && block.shown"
+    <ddp-activity-text-answer *ngIf="isTextQuestion(block) && !isEmailQuestion(block) && block.shown"
                               [block]="block"
```

Comment:
From API perspective, email question is just a text question. But it works and looks very different than a regular text question, so some special treatment.